packages/core/src/lib/lit-core.ts

```diff
@@ -837,15 +865,13 @@ export class LitCore {
 
     const data = {
       clientPublicKey: 'test',
-      challenge: params.challenge,
+      challenge,
     };
 
     return await this.sendCommandToNode({
       url: urlWithPath,
       data,
       requestId,
-    }).catch((err: NodeErrorV3) => {
-      return err;
```

Comment:
Hiding this error caused `handshakeResult` to be `err`, then placing `undefined` for all values in `keys` [here](https://github.com/LIT-Protocol/js-sdk/pull/590/files#diff-1cc39c82b504bd1b4173261252ff8fab20ec90a38134c9aa0f7d2e57094e8a89L539-L551)

Issue was that when returning an error, like the timeout one, finished everything without throwing. And this could put us in the `latestBlockhash is undefined` state